cms/djangoapps/contentstore/management/commands/add_instructor_ids.py

```diff
@@ -0,0 +1,102 @@
+"""
+Command to add UUID for Instructor.
```

Comment:
`Command created to back-populate a UUID on instructor data added to courses via Studio.`